sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -257,12 +257,13 @@ export class CASService {
       error.response?.status === HttpStatus.BAD_REQUEST &&
       !!error.response?.data[CAS_RETURNED_MESSAGES]
     ) {
-      const casKnownErrors =
-        error.response.data[CAS_RETURNED_MESSAGES].toString();
+      const casKnownErrors = error.response.data[
+        CAS_RETURNED_MESSAGES
+      ] as string;
       throw new CustomNamedError(
         "CAS Bad Request Errors",
         CAS_BAD_REQUEST,
-        casKnownErrors.split(" | ").trim(),
+        casKnownErrors.trim().split(" | "),
```

Comment:
To be clear, that was not what I meant when I said to `trim()`.

Reply:
The string "Error 1 | Error 2" should be splitter by "|".
Then the results will be "Erro 1 " and " Error 2", those should have the `trim()` applied.
I also mentioned to have one unit test about it before the ticket was delivered.